test/node/reducers/vis-state-merger-test.js

```diff
@@ -400,39 +400,6 @@ test('VisStateMerger.v1 -> mergeLayers -> toEmptyState', t => {
   t.end();
 });
 
-test('VisStateMerger.v1 -> mergeLayers -> toEmptyState', t => {
```

Comment:
this test deleted?

Reply:
oh yeah, this test was a duplicate of the one at line [370](https://github.com/keplergl/kepler.gl/pull/1239/files/a81f14147b3f2a56bcb77f38778e2d5bb6928823#diff-d07680eb0244d66267cee2d1d5f6e946f1169124d4ca0bbeabcbdd1cc9e539eeL370)